assets/js/admin/course-theme-sidebar.js

```diff
@@ -3,49 +3,147 @@
+	block.attributes.id &&
+	! block.attributes.draft;
 
 /**
  * Course Theme Sidebar component.
  */
 const CourseThemeSidebar = () => {
 	const [ theme, setTheme ] = useCourseMeta( '_course_theme' );
 
+	const globalLearningModeEnabled =
+		window.senseiSettings.sensei_learning_mode_all || false;
+	const currentPost = useSelect( ( select ) =>
+		select( 'core/editor' ).getCurrentPost()
+	);
+
+	const firstLesson = useSelect( ( select ) => {
+		const { getBlocks, getBlockAttributes } = select( 'core/block-editor' );
+		const blocks = getBlocks();
+		const courseOutline = blocks.find(
+			( block ) => block.name === courseOutlineBlockName
+		);
+
+		if ( ! courseOutline ) {
+			return {};
+		}
+
+		for ( const moduleOrLesson of getBlocks( courseOutline.clientId ) ) {
+			if ( canPreview( moduleOrLesson ) ) {
+				return getBlockAttributes( moduleOrLesson.clientId );
+			}
+
+			if ( moduleOrLesson.name === courseModuleBlockName ) {
+				for ( const lesson of getBlocks( moduleOrLesson.clientId ) ) {
+					if ( canPreview( lesson ) ) {
+						return getBlockAttributes( lesson.clientId );
+					}
+				}
+			}
+		}
+
+		return {};
+	} );
+
+	let previewUrl = '';
+	if ( firstLesson?.id && currentPost?.id ) {
+		if (
+			globalLearningModeEnabled ||
+			currentPost.meta._course_theme === SENSEI_THEME
+		) {
+			previewUrl = `/?p=${ firstLesson.id }`;
+		} else {
+			previewUrl = `/?p=${ firstLesson.id }&learn=1&${ SENSEI_PREVIEW_QUERY }=${ currentPost.id }`;
+		}
+	}
+
+	let customizerUrl = '';
+	if ( previewUrl ) {
+		customizerUrl = `/wp-admin/customize.php?url=${ encodeURIComponent(
+			previewUrl
+		) }`;
+	}
+
 	return (
 		<PluginDocumentSettingPanel
 			name="sensei-course-theme"
-			title={ __( 'Course Theme', 'sensei-lms' ) }
+			title={ __( 'Course Styles', 'sensei-lms' ) }
 		>
-			<p>
-				{ __(
-					'This does not change the theme of your site. It only applies to logged in users when viewing the course.',
-					'sensei-lms'
-				) }
-			</p>
-			<SelectControl
-				label={ __( 'Theme', 'sensei-lms' ) }
-				hideLabelFromVision
-				value={ theme }
-				options={ [
-					{
-						label: __( 'Sensei Theme', 'sensei-lms' ),
-						value: SENSEI_THEME,
-					},
-					{
-						label: __( 'WordPress Theme', 'sensei-lms' ),
-						value: WORDPRESS_THEME,
-					},
-				] }
-				onChange={ setTheme }
-			/>
+			{ globalLearningModeEnabled ? (
+				<>
+					<p className="sensei-course-theme-toggle__global-overwrite-notice">
+						<a href="/wp-admin/admin.php?page=sensei-settings#course-settings">
+							{ __(
+								'Learning Mode is enabled globally.',
+								'sensei-lms'
+							) }
+						</a>
+					</p>
+				</>
+			) : (
+				<>
+					<ToggleControl
+						className="sensei-course-theme-toggle"
+						label={
+							<>
+								<p className="sensei-course-theme-toggle__label">
+									{ __( 'Learning mode', 'sensei-lms' ) }
+								</p>
+								<p className="sensei-course-theme-toggle__description">
```

Comment:
From this part, I think it would be nice to be separate to be more semantic. Otherwise, everything would be part of the `<label>`.

ToogleControl has a [help prop](https://github.com/WordPress/gutenberg/tree/trunk/packages/components/src/toggle-control) that we could use for the rest of the content. Also, probably we don't need the `<p className="sensei-course-theme-toggle__label">`.

Reply:
Thanks for the tip. Updated. 9cd0c14eacade6e704cc491624e77b48865258e2

Reply:
Should we also remove the `__label` class from the CSS?

Reply:
Yep, forgot about that. Removed the CSS. 0ca13acf831cf54eb97e6f8648ea8a77bbc7b45f